monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1353 @@
+            legendFormat="{{job}}",
+        ),
+    ],
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+    unit="decmbytes",
+)
+
+network_io = mongodb_timeseries(
+    "Network I/O",
+    [
+        Target(
+            expr=f'{prefix}rate(mongodb_ss_network_bytes{direction}{{job=~"$job", namespace="{NAMESPACE}"}}[$__rate_interval])',
+            legendFormat=f"{{{{job}}}}: {legend}",
+        )
+        for direction, legend, prefix in [
+            ("In", "In", ""),
+            ("Out", "Out", "-"),
+        ]
+    ],
+    unit="decbytes",
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+disk_io_util = mongodb_timeseries(
+    "Disk IO Utilization (all instances)",
+    [
+        Target(
+            expr=f'sum by(pod) (rate(container_blkio_device_usage_total{{operation="Total", namespace="{NAMESPACE}", pod=~"{POD_PATTERN}"}}[$__rate_interval]))',
+            legendFormat="{{pod}}",
+        )
+    ],
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+    unit="binBps",
+)
+
+# WiredTiger Panels
+eviction_thread_created = mongodb_timeseries(
+    "Eviction thread created",
+    [
+        Target(
+            expr=f'rate(mongodb_ss_wt_cache_eviction_worker_thread_created{{job=~"$job", namespace="{NAMESPACE}"}}[$__rate_interval])',
+            legendFormat="{{pod}}",
+        )
+    ],
+    fillOpacity=0,
+    showPoints="auto",
+    unit=UNITS.OPS_PER_SEC,
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+eviction_threads_removed = mongodb_timeseries(
+    "Eviction threads removed",
+    [
+        Target(
+            expr=f'rate(mongodb_ss_wt_cache_eviction_worker_thread_removed{{job=~"$job", namespace="{NAMESPACE}"}}[$__rate_interval])',
+            legendFormat="{{pod}}",
+        )
+    ],
+    fillOpacity=0,
+    showPoints="auto",
+    unit=UNITS.OPS_PER_SEC,
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+num_eviction_threads = mongodb_timeseries(
+    "Number of active eviction threads",
+    [
+        Target(
+            expr=f'mongodb_ss_wt_cache_eviction_worker_thread_active{{job=~"$job", namespace="{NAMESPACE}"}}',
+            legendFormat="{{ job }}",
+        )
+    ],
+    fillOpacity=0,
+    showPoints="auto",
+    unit="threads",
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+wt_concurrency_tickets = mongodb_timeseries(
+    "Available Concurrency Tickets by Pod",
+    [
+        Target(
+            expr=f'sum by (pod) (mongodb_mongod_wiredtiger_concurrent_transactions_available_tickets{{txn_rw="{txn_type}", job=~"$job", namespace="{NAMESPACE}"}})',
+            legendFormat=f"{{{{pod}}}} {txn_type}",
+        )
+        for txn_type in ["read", "write"]
```

Comment:
```suggestion
        Target(
            expr=f'sum by (pod, txn_type) (mongodb_mongod_wiredtiger_concurrent_transactions_available_tickets{{txn_rw="read|write", job=~"$job", namespace="{NAMESPACE}"}})',
            legendFormat=f"{{{{pod}}}} {txn_type}",
        )
```
(and we may remove the filter on `txn_type`, if there are only 2 values)